packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -3149,3 +3150,143 @@ test('Bug 7796 missing lockedPrice', async t => {
     },
   });
 });
+
+test('Bug 7851 & no bidders', async t => {
+  const { zoe, aeth, run, rates: defaultRates } = t.context;
+
+  const rates = harden({
+    ...defaultRates,
+    liquidationPenalty: makeRatio(1n, run.brand),
+    liquidationMargin: run.makeRatio(150n),
+    mintFee: run.makeRatio(50n, 10_000n),
+  });
+  t.context.rates = rates;
+
+  const manualTimer = buildManualTimer();
+  const TEN_MINUTES = 10n * SECONDS_PER_MINUTE;
```

Comment:
Doesn't seem worth naming. If it's to be shorter we can import the constants with shorter names. Eg "MIN"